### PR TITLE
Minor refactoring and cleaning of flake8/linter suggestions

### PR DESCRIPTION
**Pull request by rly**
Just a little code and documentation cleanup

### CODE REVIEW

File: src/pynwb/__init__.py [hunk 1]
```diff
@@ -4,28 +4,24 @@
 import os.path
 from copy import deepcopy
 from warnings import warn
-
 import h5py
+from pkg_resources import resource_filename
```
Comment (oruebel) [inline src/pynwb/__init__.py]: This was probably intentionally a function-level import because you don't want to pull it into the pynwb module

Comment (rly) [reply src/pynwb/__init__.py]: Ohhh I see.

Comment (rly) [reply src/pynwb/__init__.py]: This code is always run though (see line 102), so wouldn't it be pulled into the pynwb module anyway?

Comment (rly) [reply src/pynwb/__init__.py]: Never mind. I tested it and you are right. It is not pulled in if it is in the function. Will change.

**Review by oruebel — APPROVED** [review]

**Pull request merged**